Framework/Geometry/inc/MantidGeometry/Crystal/PeakShape.h

```diff
@@ -46,6 +47,10 @@ class DLLExport PeakShape {
   virtual int algorithmVersion() const = 0;
   /// Shape name
   virtual std::string shapeName() const = 0;
+  /// For selecting different radius types.
+  enum RadiusType { Radius = 0, OuterRadius = 1, InnerRadius = 2 };
```

Comment:
We might want to consider using a strongly typed enum. At the interface boundaries, i.e. where `int`s are passed , we could have a simple translation function from `int` to the declared enum.

We could do this as a second step or for maintenance.